libs/scripts/scripts.js

```diff
@@ -44,8 +44,6 @@ export async function loadLCP(blocks) {
   }
 }
 
-function loadDelayed() {}
```

Comment:
Did you mean to remove this? Causes console error a few lines down at `loadDelayed();`

Reply:
Yes. I did. A) knowing that @amauch-adobe was bringing it back, but more importantly B) It was making my PR for a README look bad on codecov. 😂